src/Common/tests/System/PlatformDetection.cs

```diff
@@ -18,9 +18,10 @@ public static partial class PlatformDetection
         // do it in a way that failures don't cascade.
         //
 
-        public static bool IsUap => IsWinRT || IsNetNative;
+        public static bool IsUap => IsInAppContainer || IsNetNative;
```

Comment:
> I guess this was wrongly implemented? 

I think so. 

> this wasn't added in this PR, IsUap already exists in master...)

right but we need to fix it. feel free to fix it in other PR if you like or fix it with this PR.